types/jquery-typeahead/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Please re-enable all these lint rules.

Reply:
Hi @DanielRosenwasser, thank you for your comment. The issue is fixed.